kinto/__main__.py

```diff
@@ -93,6 +99,4 @@ def main(args=None):
             pserve_argv.append('--reload')
         pserve.main(pserve_argv)
 
-
-if __name__ == "__main__":
-    main()
```

Comment:
I was using it to test locally using `python kinto/__main__.py`, which is a little easier than running `pip install` every time I make a change..